doc/implementdoc/equ-commodity.rst

```diff
@@ -403,17 +403,17 @@ and calculated by the following code fragment:
 
 **Global CO2 Budget Rule**: The constraint global CO2 budget rule applies to
 the whole energy system over the entire modeling horizon, that is to say it
-applies to every support timeframe, site and timestep. This constraints
-restricts the total amount of CO2 to environment. The constraint states that
+applies to every support timeframe, site and timestep. This constraint
+restricts the total amount of CO2 emission to environment. The constraint states that
 the sum of released CO2 across all support timeframe :math:`y\in Y`, sites
-:math:`v\in V` and timesteps :math:`t \in t_m` must be less than or equal to
+:math:`v\in V` and timesteps :math:`t \in T_m` must be less than or equal to
 the parameter maximum global CO2 emission budget
 :math:`\overline{\overline{L}}_{CO_{2},y}`, where the amount of released CO2 in
 a single support timeframe :math:`y` in a single site :math:`v` and at a single
 timestep :math:`t` is calculated by the product of the commodity balance of
 environmental commodities :math:`\mathrm{CB}(y,v,CO_{2},t)` and the parameter
 weight :math:`w`. This constraint is skipped if the value of the parameter
-:math:`\overline{\overline{L}}_{CO_{2}}` is set to ``inf``. The mathematical
+:math:`\overline{\overline{L}}_{CO_{2},y}` is set to ``inf``. The mathematical
```

Comment:
Global CO2 Budget is '\overline{\overline{L}}_{CO_{2}}'?

Reply:
Hi, Global CO2 Budget is not a function of modeled years (y). It corresponds to "CO2 Emission Budget for modeling horizon".

![grafik](https://user-images.githubusercontent.com/30022650/94809036-4e909800-03f2-11eb-825e-cb1f4722e992.png)

Reply:
Yes. So line 416 should not be updated with addition of 'y'.